packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1091,6 +1101,10 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   ///
   /// By default, the cursor should be painted on top for iOS platforms and
   /// underneath for Android platforms.
+  ///
+  /// The hierarchy of the selection box follows the cursor mean that selection
```

Comment:
How about changing the first line from 
> "If the cursor should be painted on top of the text or underneath it."

to something like:

> "If the cursor and the text selection highlight should be painted of top of the text or underneath it"?

Reply:
done

Reply:
It's still a bit unclear as to what this is referring to. Adding "text selection highlight" to the summary should be good enough, it's probably ok to not further explain the (iOS-specific) details.

Reply:
Do you mind removing this paragraph?